src/govuk/components/exit-this-page/template.test.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env jest */
+
+const { render } = require('govuk-frontend-helpers/nunjucks')
+const { axe } = require('govuk-frontend-helpers/tests')
+const { getExamples } = require('govuk-frontend-lib/files')
+
+describe('Exit this page', () => {
+  let examples
+
+  beforeAll(async () => {
+    examples = await getExamples('exit-this-page')
+  })
+
+  describe('default example', () => {
+    it('passes accessibility tests', async () => {
+      const $ = render('exit-this-page', examples.default)
+
+      const results = await axe($.html())
+      expect(results).toHaveNoViolations()
+    })
+
+    it('renders the default example', () => {
+      const $ = render('exit-this-page', examples.default)
+      const $button = $('.govuk-exit-this-page').find('.govuk-button')
+
+      expect($button.hasClass('govuk-button--warning')).toBeTruthy()
+      expect($button.text()).toContain('Exit this page')
+      expect($button.attr('href')).toBe('https://www.gov.uk')
+    })
+  })
+
+  describe('Custom options', () => {
+    it('renders with text', () => {
+      const $ = render('exit-this-page', examples.testing)
+      const $button = $('.govuk-exit-this-page').find('.govuk-button')
+
+      expect($button.text()).toContain('Exit this test')
+    })
+
+    it('renders with a current tab URL', () => {
```

Comment:
What does this mean?

Reply:
Stale wording from when we did fancy tab stuff on click. Have since updated it.